tests/test_matcher_email.py

```diff
@@ -165,6 +165,85 @@ def test_match_with_blacklist(self):
         result = matcher.match(jane_rae, jrae)
         self.assertEqual(result, False)
 
+    def test_match_with_blacklist_not_strict(self):
+        """Test match when there are entries in the blacklist and strict mode is False"""
+
+        # Let's define some identities first
+        jsmith = UniqueIdentity(uuid='jsmith')
+        jsmith.identities = [Identity(name='John Smith', email='jsmith@example.com', source='scm'),
+                             Identity(name='John Smith', source='scm'),
+                             Identity(username='jsmith', source='scm'),
+                             Identity(email='', source='scm')]
+
+        john_smith = UniqueIdentity(uuid='js')
+        john_smith.identities = [Identity(name='J. Smith', username='john_smith', source='scm'),
+                                 Identity(username='john_smith', source='scm'),
+                                 Identity(name='Smith. J', source='mls'),
+                                 Identity(name='Smith. J', email='JSmith@example.com', source='mls')]
+
+        jrae = UniqueIdentity(uuid='jrae')
+        jrae.identities = [Identity(name='Jane Rae', source='scm', uuid='jrae'),
+                           Identity(name='Jane Rae Doe', email='jane.rae@example.net', source='mls', uuid='jrae'),
+                           Identity(name='jrae', source='scm', uuid='jrae'),
+                           Identity(email='JRAE@example.net', source='scm', uuid='jrae')]
+
+        jane_rae = UniqueIdentity(uuid='Jane Rae')
+        jane_rae.identities = [Identity(name='Jane Rae', source='scm', uuid='Jane Rae'),
+                               Identity(email='jrae@example.net', source='mls', uuid='Jane Rae')]
+
```

Comment:
If the strict mode is `False` there should be at least one example where the field `email` is not a strict email address.